packages/api/src/task/scheduler.ts

```diff
@@ -174,6 +177,8 @@ export class TaskScheduler {
       status: {
         ...task.status,
         phase: "completed",
+        progress: undefined,
+        step: undefined,
```

Comment:
Need to figure out how to do this more cleanly instead of clearing the fields on every fn place we update task or asset status. Maybe some built-in logic in `updateTask` and `updateAsset`? Or always making the updates with all fields specified, instead of doing the `...` merge? Open to suggestions.

Reply:
Thinking with myself, I think I'm gonna go with stopping the merge madness and just declare the full expected status on every update. Easier to reason about.

Reply:
Done!